spec/root_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'index route', type: :system do
```

Comment:
This looks like a cross between controller/view/request/feature spec. Familiarise yourself with the following guidelines to create tests that closely aligns with these examples. 

https://relishapp.com/rspec/rspec-rails/docs/controller-specs
https://relishapp.com/rspec/rspec-rails/v/6-0/docs/view-specs/view-spec
https://relishapp.com/rspec/rspec-rails/v/6-0/docs/feature-specs/feature-spec
https://relishapp.com/rspec/rspec-rails/v/6-0/docs/request-specs/request-spec

Reply:
Changed the test, does it look better now?